impl/src/main/java/io/kroxylicious/testing/kafka/common/Utils.java

```diff
@@ -151,6 +153,14 @@ public static void awaitExpectedBrokerCountInClusterViaTopic(Map<String, Object>
                         final Boolean isQuorate = promise.getNow(false);
                         if (isQuorate) {
                             admin.deleteTopics(Set.of(CONSISTENCY_TEST));
+                            for (int i = 0; i < admin.describeCluster().nodes().get().size(); i++) {
+                                ConfigResource resource = new ConfigResource(ConfigResource.Type.BROKER, String.valueOf(i));
+                                Config configs = admin.describeConfigs(List.of(resource)).all().get().get(resource);
+                                if (!configs.get("delete.topic.enable").value().equals("false")) {
```

Comment:
I think we should move the if test around the whole delete shouldn't we? There is no point doing any of this if `delete.topic.enable == false`.

Reply:
If we remove this if clause, the test `ParameterExtensionTest.brokerConfigs` fails, because it forces to set that condition `delete.topic.enable == false` and it will never delete the topic so we will receive an exception when the timeout ends.

Reply:
I meant move the test for the config property to be before the attempt to delete so that we don't try to delete if its set to false.

Reply:
Yeap, sorry I misunderstood the point of your comment. Totally agree

Reply:
All good 😀

Reply:
I tried to leave as it was, deleting the topics anyway. In that way we don't modify the behavior of the deletion action (if kafka is implemented well, the deletion won't be done if `delete.topic.enable == false`) and then we wait for the deletion only when needed.

Reply:
I'm sure Kafka will do the right thing, however I was looking to avoid the RPC call if we knew it wouldn't work. Having said that given we need to make another RPC call to work that out its possibly moot.

I'll have a play tomorrow my time with blocking on the future even if its just to understand why one gets the errors you observed.

Reply:
Right. I put it inside the if clause.